LightningRod/app/src/main/java/com/elysium/lightningrod/MainActivity.java

```diff
@@ -0,0 +1,111 @@
+package com.elysium.lightningrod;
+
+import android.content.Context;
+import android.net.ConnectivityManager;
+import android.net.NetworkInfo;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.elysium.lightningrod.Model.City;
+
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+public class MainActivity extends AppCompatActivity {
+
+    EditText editText;
+    Button button;
+    TextView cityView, descriptionView, humidityView, pressureView, temperatureView;
+    String cityName;
+
+    public static final String API_KEY = "a3805d0a057d5e356d6aea4d6a30e4d1";
```

Comment:
Best practice is to put constants at the top of the class, before member variables